sources/packages/web/src/components/common/ProgramDetailHeader.vue

```diff
@@ -0,0 +1,135 @@
+      <header-title-value title="Institution name"
+        ><template #value
+          ><a
+            v-if="institutionId"
+            @click="goToInstitution()"
+            class="label-bold-primary"
+          >
+            {{ educationProgram.institutionName }}
+          </a>
+          <span v-else>{{ educationProgram.institutionName }}</span>
+        </template></header-title-value
+      >
+      <div class="mx-2 vertical-divider"></div>
+      <header-title-value
+        title="Submitted"
+        :value="
+          educationProgram.submittedOn
+            ? originalDateOnlyLongString(educationProgram.submittedOn)
+            : '-'
+        "
+      />
+    </div>
+    <div
+      class="row mt-1"
+      v-if="ApprovalStatus.approved === educationProgram.approvalStatus"
+    >
+      <header-title-value
+        title="Approved by"
+        :value="approvedBy ?? '-'"
+        v-if="approvedBy"
+      />
+      <div class="mx-2 vertical-divider" v-if="approvedBy"></div>
+      <header-title-value
+        title="Approved"
+        v-if="educationProgram.approvedOn"
+        :value="
+          educationProgram.approvedOn
+            ? originalDateOnlyLongString(educationProgram.approvedOn)
+            : '-'
+        "
+      />
+      <div
+        class="mx-2 vertical-divider"
+        v-if="educationProgram.effectiveEndDate"
+      ></div>
+      <header-title-value
+        v-if="educationProgram.effectiveEndDate"
+        title="Effective end date"
+        :value="
+          educationProgram.effectiveEndDate
+            ? dateOnlyLongString(educationProgram.effectiveEndDate)
+            : '-'
+        "
+      />
+    </div>
+    <div
+      class="row mt-1"
+      v-if="ApprovalStatus.denied === educationProgram.approvalStatus"
+    >
+      <header-title-value title="Denied by" :value="deniedBy" />
+      <div class="mx-2 vertical-divider"></div>
+      <header-title-value
+        title="Denied"
+        :value="
+          educationProgram.deniedOn
+            ? originalDateOnlyLongString(educationProgram.deniedOn)
+            : '-'
+        "
+      />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { EducationProgramData, ApprovalStatus } from "@/types";
+import HeaderTitleValue from "@/components/generic/HeaderTitleValue.vue";
+import { useFormatters } from "@/composables";
+import { computed } from "vue";
+import { useRouter } from "vue-router";
+import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
+
+export default {
+  components: { HeaderTitleValue },
+  props: {
+    educationProgram: {
+      type: Object,
+      required: true,
+      default: {} as EducationProgramData,
+    },
+    institutionId: {
+      type: Number,
+      required: false,
+    },
+  },
+  setup(props: any) {
+    const router = useRouter();
+    const { originalDateOnlyLongString, dateOnlyLongString } = useFormatters();
+    const getFullName = (firstName: string, lastName: string) => {
```

Comment:
Full names must consider also mononynous names. Please take a look in the method already present in the API.
Until now all the name formatting are coming ready from the API. If the names are going to be formatted in the Vue application it must follow the same method from the API.
This formatter seems to be formatting "LastName, FirstName", what is different from what we are doing in the API. I believe that it would be good to double check with UI/UX if they need be different and avoid creating two different way of formatting names.